standalone/src/handlers.rs

```diff
@@ -229,9 +229,6 @@ pub async fn ingest(
     let mut graph_ops = GraphOps::new();
     graph_ops.connect().await?;
 
-    info!("Clearing old data...");
-    graph_ops.graph.clear().await?;
```

Comment:
you still need to clear... but only the one you ingested

so use file STARTS WITH